addons/metrics-server/README.md

```diff
@@ -76,4 +76,4 @@ behavior:
 - `--kubelet-preferred-address-types`: the order in which to consider
   different Kubelet node address types when connecting to Kubelet.
   Functions similarly to the flag of the same name on the API server.
-s
\ No newline at end of file
+s
```

Comment:
Any reason for this change?

Reply:
@rdrgmnzs Reviewed and removed the newline. The `s` was already in there and it looked like a typo, so I removed it. Thank you for your patience!